demo/kserve/scripts/install/1.prerequisite-operators.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
would be better to rename the bash script to `1-...` instead using 1.

Reply:
accept this :)